crates/sui-verifier/src/entry_points_verifier.rs

```diff
@@ -222,15 +218,12 @@ fn verify_return_type(
 
 fn verify_param_type(
     view: &BinaryIndexedView,
-    module_id: &ModuleId,
     function_type_args: &[AbilitySet],
     param: &SignatureToken,
 ) -> Result<(), String> {
     // Only `sui::sui_system` is allowed to expose entry functions that accept a mutable clock
     // parameter.
-    if module_id != &ModuleId::new(SUI_FRAMEWORK_ADDRESS, SUI_SYSTEM_MODULE_NAME.to_owned())
-        && is_mutable_clock(view, param)
-    {
+    if is_mutable_clock(view, param) {
```

Comment:
What happened to this case?

Reply:
That function was moved out of sui system module and made private so not an entry fun anymore